adapters/platform_win32.c

```diff
@@ -38,10 +39,20 @@ int platform_init(void)
     }
     else
     {
+        result = 0;
+#ifndef DONT_USE_UPLOADTOBLOB
+        if (HTTPAPIEX_Init() == HTTPAPIEX_ERROR)
+        {
+            LogError("HTTP for upload to blob failed on initialization.");
+            result = MU_FAILURE;
+        }
+#endif /*DONT_USE_UPLOADTOBLOB*/
 #ifdef USE_OPENSSL
-        tlsio_openssl_init();
+        if (result = 0)
```

Comment:
>= [](start = 19, length = 1)

should be `==`

Reply:
Nice catch, tks!

---
In reply to: [319609282](https://github.com/Azure/azure-c-shared-utility/pull/366#discussion_r319609282) [](ancestors = 319609282)